README.md

```diff
@@ -1 +1,152 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [The Capstone Project] <a name="Capstone Project"></a>
+
+**[Capstone]** is a project to build a website to show a future Environmental Protection Global submit
+
+## 🛠 Built With <a name="built-with">Visual Studio</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Visual Studio</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Github pages</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>No database was  used</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">NA</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Mobile version with Mobile menu]**
+- **[A Navbar with a banna]**
+- **[Speakers section dynamically generated by JS]**
+- **[Desktop Version]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://enocol.github.io/capstone/about.html)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+- create a local directory on your computer.
+- Open a Terminal and run the following code
+
+git clone https://github.come/enocol/capstone.git
+
+Note: You must have git running on your computer.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+### Install
+
+Install this project with:
+
+### Usage
+
+To run the project, execute the following command:
+open the project in your refered code Editors and run a local live server.
+
+### Run tests
+
+To run tests, run the following command:
+
+### Deployment
+
+You can deploy this project using:
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Enoh Collins**
+
+- GitHub: [@githubhandle](https://github.com/enocol)
+- Twitter: [@twitterhandle](https://twitter.com/enocol)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/enocol)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Please 🙏🏻 adhere to the project's general requirements properly according to which you should have to give the credit to the designer **Cindy Shin**, which shows that you are following the design usage terms and conditions. A demo video is attached below.

[Screencast from 08-07-23 04:11:15 PM IST.webm](https://github.com/ulises2607/cayafate-project/assets/66476329/92684188-88cf-4dd7-98a6-f14780f05722)

- [ ] Please also add some valid, and valuable information in your support section.

Reply:
I think the video above is the only information I have received from you. The rest is outwardly unprofessional. Since you have used a figma design to evaluate my project, I have no option but to stop at this stage. Thank you.